apps/prairielearn/elements/pl-file-download/pl-file-download.py

```diff
@@ -38,36 +79,27 @@ def render(element_html, data):
     )
 
     # Get base url, which depends on the type and directory
-    if file_type == "static":
-        if file_directory == "clientFilesQuestion":
-            base_url = data["options"]["client_files_question_url"]
-        elif file_directory == "clientFilesCourse":
-            base_url = data["options"]["client_files_course_url"]
-        else:
-            raise ValueError(
-                'directory "{}" is not valid for type "{}" (must be "clientFilesQuestion" or "clientFilesCourse")'.format(
-                    file_directory, file_type
-                )
-            )
-    elif file_type == "dynamic":
+    if file_type is FileType.STATIC:
+        # Get directory (default is clientFilesQuestion)
```

Comment:
Micro-nit, this strikes me as a comment that doesn't say anything not already expressed in the code.
```suggestion
```